code/src/App.js

```diff
@@ -1,9 +1,22 @@
 import React from 'react'
+import { Provider } from 'react-redux'
+import { combineReducers, configureStore } from '@reduxjs/toolkit'
```

Comment:
Change this line (line 3) to this:

`import { compose, createStore, combineReducers } from "@reduxjs/toolkit";`